cloud/cloud-multi-node.md

```diff
@@ -7,14 +7,24 @@ tags: [cluster, distributed hypertables]
 ---
 
 # Multi-node on Timescale Cloud
+
 If you have a larger workload, you might need more than one TimescaleDB
 instance. TimescaleDB multi-node allows you to run and manage multiple
 instances, which can give you faster data ingest, and more responsive and
 efficient queries for many large workloads.
 
 This section shows you how to use multi-node on Timescale Cloud. You can also
-use multi-node on [Managed Service for TimescaleDB][multinode-mst], or
-[self-hosted TimescaleDB][multinode-timescaledb].
+set up multi-node on [self-hosted TimescaleDB][multinode-timescaledb].
+
+<highlight type="important">
+Multi-node in Timescale Cloud is an early access feature. If you'd like to use
+multi-node, first contact the Timescale customer support team to discuss your
+use case. The team can help you understand the trade-offs and whether it's a
```

Comment:
I'm not sure I like the trade-off wording. Could we just say something like "The team can help you understand if multi-node is a good fit for your needs"?